app/client/src/features/events/LiveFeedback/LiveFeedbackAuthor.tsx

```diff
@@ -41,7 +42,12 @@ export function LiveFeedbackAuthor({ fragmentRef, ...props }: LiveFeedbackAuthor
     );
     return (
         <CardHeader
-            title={<Typography>{authorData.createdBy?.firstName ?? 'Unknown User'}</Typography>}
+            avatar={
+                <Avatar>
+                    {authorData.createdBy?.firstName ? authorData.createdBy?.firstName[0] : 'U'}
+                </Avatar>
+            }
+            title={<Typography>{(authorData.createdBy?.firstName + ' ' + authorData.createdBy?.lastName) ?? 'Unknown User'}</Typography>}
```

Comment:
This could potentially result in `David undefined`.  I would put this into its own function so that way we can more robustly cover all cases.